scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -952,9 +949,9 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn,
             line = ""
 
         handler_time = time.time() - start_time
-        logger.debug(f"QEMU ({pid}) complete ({out_state}) after {handler_time} seconds")
+        logger.debug(f"QEMU ({pid}) complete ({_status} / {_reason}) after {handler_time} seconds")
```

Comment:
To be consistent with https://github.com/zephyrproject-rtos/zephyr/blob/a70bf0e488c5327ad67512c2854987d113318c79/scripts/pylib/twister/twisterlib/reports.py#L471, it should be more akin to: `logger.debug(f"QEMU ({pid}) complete with status: {_status} ({_reason}) after {handler_time} seconds")`